console/src/app/pages/users/user-list/user-table/user-table.component.ts

```diff
@@ -194,6 +194,9 @@ export class UserTableComponent implements OnInit {
         case 'username':
           sortingField = UserFieldName.USER_FIELD_NAME_USER_NAME;
           break;
+        case 'preferredLoginName':
+          sortingField = UserFieldName.USER_FIELD_NAME_USER_NAME;
```

Comment:
I think this should be Preferred loginname instead of username?

Reply:
i think that will do for the moment. @peintnermax what do you think?

Reply:
I think this is fine for the moment.. The result rows might not be expected but the sorting could help 👍